content/_help/verify-your-identity/how-to-verify-your-identity._zh.md

```diff
@@ -63,7 +63,8 @@ Login.gov 是一个安全的、遵守数据保护最高标准的政府网站。
 3. 输入你的社会保障号码，然后点击“继续”按钮。
 4. 核对一下你到此为止上传的信息都是正确的，然后点击“继续”按钮。
 5. 输入你的主电话号码。这是身份验证的最后一步。   
-   * 如果你没有美国电话号码或不能通过电话验证，点击“通过普通邮件验证地址”。你得等 5 到 10 天才能收到邮寄的信，收到后请按照其中指示输入验证码。
+   * 在某些情况下，您可能可以选择“改用邮局信件验证您的地址”。
+   * 如果您没有通过邮局信件验证的选项，则必须通过电话验证才能成功验证您的身份。
```

Comment:
Mercifully, the Chinese Markdown file was already going in numeric order. This is the one where I would have been so lost without it.